lib/content-services/src/lib/tree/components/tree.component.html

```diff
@@ -44,7 +44,7 @@
                 [attr.data-automation-id]="'node_' + node.id"
                 *matTreeNodeDef="let node"
                 matTreeNodePadding
-                [adf-context-menu]="contextMenuOptions"
+                [adf-context-menu]="getContextMenuOptions"
```

Comment:
Do you tried `OnPush` change detection to avoid mulitple calls? https://hyland.atlassian.net/wiki/spaces/TECH/pages/1293321363/FE+code+best+practices#Use-OnPush-Change-Detection-Strategy

Reply:
This would also work but as it was on component level I was not sure, if I could update it.